src/packaging/kpatch/commit_to_overlay.sh

```diff
@@ -401,7 +430,8 @@ for change in ${COMMIT_SHAS[@]}; do
   # Build the tree as normal
   build_log_file=$(mktemp /tmp/overlay_build_log.XXXXX)
   echo -en "Building the VDO tree... "
```

Comment:
Since we're no longer building the entire tree, I feel that this message should be modified to indicate which portions are being built.

Reply:
I've added updated out out for each command, and update the section comment as well.